addons/api/addon/handlers/client-daemon-handler.js

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+/* global __electronLog */
```

Comment:
What is the purpose of this comment?

Reply:
It's to let eslint know that we're using a global variable